bin/xbundle_convert

```diff
@@ -46,7 +47,9 @@ def main():
         options['force_studio_format'] = True
 
     if args['test']:
-        run_tests()
+        base_dir = os.path.dirname(os.path.realpath(os.path.join(
+            __file__, "..", "tox.ini")))
```

Comment:
This can just be:

```python
base_dir = os.path.dirname(os.path.realpath( __file__))
```

This worked for me:

```diff
diff --git bin/xbundle_convert bin/xbundle_convert
index 64b0a49..a83312e 100755
--- bin/xbundle_convert
+++ bin/xbundle_convert
@@ -49,6 +49,8 @@ def main():
     if args['test']:
         base_dir = os.path.dirname(os.path.realpath(os.path.join(
             __file__, "..", "tox.ini")))
+        base_dir2 = os.path.dirname(os.path.realpath( __file__))
+        assert base_dir == base_dir2
         check_call(["tox"], cwd=base_dir)
         return
```